README.md

```diff
@@ -15,6 +15,15 @@ The Timestream JDBC driver is distributed as two JAR files.
 ### JAR With All Dependencies Shaded Except SLF4J (amazon-timestream-jdbc-\<version\>-jar-with-all-dependencies.jar)
 This is the shaded JAR which bundles up all the required dependencies and can be used directly in Java projects and external applications, without explicitly adding external dependencies. This JAR does not relocate the classpath of SLF4J, allowing logging to be enabled in external applications.
 
```

Comment:
To fellow code reviewers: for this PR, the files to focus on are the markdown files (README and set up files), please check if they load images properly and the links work. This is optional since they are already checked in separate PRs. Thank you!